compiler/test/dotty/tools/dotc/reporting/ErrorMessagesTests.scala

```diff
@@ -1638,4 +1638,24 @@ class ErrorMessagesTests extends ErrorMessagesTest {
         message.msg
       )
     }
+
+  @Test def traitParametersUsedAsParentPrefix() =
+    checkMessagesAfter(RefChecks.name) {
+      """
+        |class Outer {
+        |   trait Inner
+        |   trait Test(val outer: Outer) extends outer.Inner
+        |}
+        |""".stripMargin
+    }.expect {
+      (ictx, messages) =>
+        implicit val ctx: Context = ictx
+        val TraitParameterUsedAsParentPrefix(typ) :: Nil = messages
+        assertEquals(1, messages.size)
```

Comment:
Isn't this redundant due to the match above?

Reply:
Yes it is! Thanks for pointing that out, I'll remove it